02_configure_host.sh

```diff
@@ -227,22 +227,8 @@ done
 
 # Support for building local images
 for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
-  BRANCH_IMAGE_VAR="${IMAGE_VAR}_BRANCH"
   IMAGE="${!IMAGE_VAR}"
-  BRANCH="${!BRANCH_IMAGE_VAR:-main}"
-
-  # Is it a git repo?
-  if [[ "$IMAGE" =~ "://" ]] ; then
```

Comment:
why removing support for cloning from a git repo ?

Reply:
It was a duplication, dev-env still supports git repo cloning but now both CRD customization and local image building will rely on the repo cloning process done in 02 script in the clone_repo functions. Previously in case of local builds we had situations where dev-env cloned everything 2 times into different directories. 

This change was prepared in  https://github.com/metal3-io/metal3-dev-env/pull/1083 PR.